arduino/src/main.cpp

```diff
@@ -2,12 +2,16 @@
 #include <WiFi.h>
 #include <HTTPClient.h>
 #include <ArduinoJson.h>
+#define LED  33

```

Comment:
Myślałem, że na płytce są 2 LEDY

Reply:
A tak poza tym to korzystasz także z innych pinów. Np do przycisków

Reply:
> Myślałem, że na płytce są 2 LEDY

są dwa LEDy ale jeden jest wpięty do pina który stale przesyła 3,3 Volta do diody i nie trzeba go programować

Reply:
No to pozostaje kwestia pina przycisku.   
Chodzi o to, że w kodzie nigdzie nie powinny znajdować się tzw magiczne wartości

Zajrzyj tutaj, abyś wiedział o co mi chodzi: https://blog.webdevsimplified.com/2020-02/magic-numbers/